test/parallel/test-crypto-fips.js

```diff
@@ -0,0 +1,208 @@
+'use strict';
+var common = require('../common');
+var assert = require('assert');
+var spawn = require('child_process').spawn;
+var path = require('path');
+
+if (!common.hasCrypto) {
+  console.log('1..0 # Skipped: missing crypto');
+  return;
+}
+
+const EXIT_FAILURE = -1;
+const FIPS_ENABLED = 1;
+const FIPS_DISABLED = 0;
+const FIPS_ERROR_STRING = 'Error: Cannot set FIPS mode';
+const OPTION_ERROR_STRING = 'bad option';
+const CNF_FIPS_ON = path.join(common.fixturesDir, 'openssl_fips_enabled.cnf');
+const CNF_FIPS_OFF = path.join(common.fixturesDir, 'openssl_fips_disabled.cnf');
+var num_children_spawned = 0;
+var num_children_ok = 0;
+const child_responses = {};
```

Comment:
Is there any point in declaring it this way? I think it could live below in `testHelper`'s closure. Also it could be just two vars `stdout = ''` and `stderr = ''`.